packages/pn-commons/src/pages/AccessDenied.tsx

```diff
@@ -31,17 +33,21 @@ const AccessDenied: React.FC<Props> = ({
     <Stack
       direction="column"
       alignItems="center"
-      my={4}
+      my={6}
```

Comment:
Here and down, we override the original style. This change also the style of the AccessDenied used when user is not logged in.
That It is ok?